src/saucelabs/job.js

```diff
@@ -1,9 +1,19 @@
+import wd from 'wd';
 import Promise from 'pinkie';
 import SaucelabsRequestAdapter from './request';
-import wait from '../utils/wait';
 
-const CHECK_RESULTS_TIMEOUT = 1000 * 20;
-const MAX_RESTART_COUNT     = 3;
+
+const MAX_RESTART_COUNT        = 3;
```

Comment:
Let's rename to MAX_JOB_RESTART_COUNT, because we have several 'restartable' things here (browser initing, page opening, testsRunning)